build.gradle

```diff
@@ -7,38 +7,32 @@ enum VersionType {
 }
 
 ext {
-    graalRevision = getGraalRevision()
-    project.logger.info("Graal revision: $graalRevision")
-
-    if (version.endsWith("-SNAPSHOT")) {
-        versionType = VersionType.SNAPSHOT
-    } else if (version.matches(/.+\-RC[0-9]+$/)) {
-        versionType = VersionType.RELEASE_CANDIDATE
-    } else {
-        versionType = VersionType.GA_RELEASE
-    }
-}
-
-allprojects {
-    apply plugin: 'java'
-
     apply from: rootProject.file("versions.gradle")
```

Comment:
Can you move this out of the `ext` as well

Reply:
I'll also move the enum, not really needed there